src/lib/utils/__tests__/fetcher.test.ts

```
@@ -0,0 +1,458 @@
+    // 環境変数をリセット
+    delete process.env.NEXT_PUBLIC_DEV_MODE
+  })
+
+  afterEach(() => {
+    // 環境変数をクリア
+    delete process.env.NEXT_PUBLIC_DEV_MODE
+  })
+
+  describe('fetcher 関数テスト', () => {
+    const testUrl = 'https://api.example.com/data'
+    const mockResponseData = { id: 1, name: 'Test Data' }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)